python/tools/create_inbox_message/create_inbox_message.py

```diff
@@ -0,0 +1,33 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+
+import ftrack_api
+
+session = ftrack_api.Session()
+
+recipients_user_names = ['<username>']
```

Comment:
@seebergs maybe better use a query to get any other user to pass in to send the message , so can be run and tested by the users.

Reply:
Can you explain this in more detail? You mean not providing a list of usernames, rather something more generic?

Reply:
yes, simply so users can run the code as first thing without having to change anything.
It should be matter of picking any other users which is not the one sending the message.